Modules/Models/Configuration.swift

```diff
@@ -334,3 +339,41 @@ public extension Dictionary where Key == String, Value == URL {
     return Dictionary(uniqueKeysWithValues: values)
   }
 }
+
+public extension Dictionary where Key == String, Value == String {
+
+    /// default values for countries
+  static var defaultCountries: [String: String] {
+    let values = [
+                  ("AT","AUSTRIA"),
+                  ("BE","BELGIO"),
+                  ("BG","BULGARIA"),
+                  ("CY","CIPRO"),
+                  ("HR","CROAZIA"),
+                  ("DK","DANIMARCA"),
+                  ("EE","ESTONIA"),
+                  ("FI","FINLANDIA"),
+                  ("DE","GERMANIA"),
+                  ("GB","GRAN BRETAGNA"),
+                  ("EL","GRECIA"),
+                  ("IE","IRLANDA"),
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: Modules/Models/Configuration.swift;
  Line: 359;
-->

- :warning: swiftlint (0.39.2): There should be no space before and one after any comma. (comma)